app/javascript/controllers/selection_controller.js

```diff
@@ -81,4 +96,13 @@ export default class extends Controller {
       outlet.setDisabled(count);
     });
   }
+
+  #updateSelectAllCB(total) {
```

Comment:
Can I get you to update this method name for clarity?

Reply:
Updated in bd016c88ca3108a6bbf14b9c63fb2bb9bba44bdd